From 8cb7a71784abded19ddfb68b5a79e5554081be77 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Wed, 12 Feb 2020 13:23:38 -0600
Subject: [PATCH 037/120] arm64: dts: agilex: correct service layer driver's
 compatible value

commit da06d397152692036fdcf9966c82eea02902177c from
https://github.com/altera-opensource/linux-socfpga.git

[upstream commit fe8b9530d2788643bccdd1b7476845f3ce465682]

Correct the compatible property value for Intel Service Layer driver
on Intel Agilex SoC platform.

Signed-off-by: Richard Gong <richard.gong@intel.com>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/intel/socfpga_agilex.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
index 97701aad0cf8..b77cdcffb6b1 100644
--- a/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
+++ b/arch/arm64/boot/dts/intel/socfpga_agilex.dtsi
@@ -619,7 +619,7 @@
 
 		firmware {
 			svc {
-				compatible = "intel,stratix10-svc";
+				compatible = "intel,agilex-svc";
 				method = "smc";
 				memory-region = <&service_reserved>;
 
-- 
2.31.1

